src/components/MultiSelect/MultiSelectListbox.tsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import {
+  Box,
+  ListItem,
+  UnorderedList,
+  useMultiStyleConfig,
+} from "@chakra-ui/react";
+import Checkbox from "./../Checkbox/Checkbox";
+import { MultiSelectItem } from "./MultiSelectTypes";
+import { MultiSelectProps } from "./MultiSelect";
+import MultiSelectMenuButton from "./MultiSelectMenuButton";
+import { useSelect } from "downshift";
+
+type MultiSelectListboxProps = Omit<MultiSelectProps, "onChange"> & {
+  onChange: (selectedItem: MultiSelectItem, id: string) => void;
+};
+
+function MultiSelectListbox({
```

Comment:
Add a comment block for this specific variant's functionality.

Reply:
fixed